### PR TITLE
[CELEBORN-647][BUG] Fix potential NPE when remove push status

### PR DESCRIPTION
**Pull request by AngersZhuuuu**
### What changes were proposed in this pull request?
Fix potential NPE when remove push status

### CODE REVIEW

**Comment by AngersZhuuuu** [comment]
ping @RexXiong

File: common/src/main/java/org/apache/celeborn/common/write/InFlightRequestTracker.java [hunk 1]
```diff
@@ -62,7 +62,12 @@ public void addBatch(int batchId, String hostAndPushPort) {
 
   public void removeBatch(int batchId, String hostAndPushPort) {
     Set<Integer> batchIdSet = inflightBatchesPerAddress.get(hostAndPushPort);
-    batchIdSet.remove(batchId);
+    // TODO: Need to debug why batchIdSet will be null.
+    if (batchIdSet != null) {
+      batchIdSet.remove(batchId);
+    } else {
+      logger.error("Batch id set of " + hostAndPushPort + " is null.");
```
Comment (pan3793) [inline common/src/main/java/org/apache/celeborn/common/write/InFlightRequestTracker.java:69]:
```suggestion
      logger.warn("BatchIdSet of {} is null.", hostAndPushPort);
```

File: common/src/main/java/org/apache/celeborn/common/write/InFlightRequestTracker.java [hunk 2]
```diff
@@ -171,6 +176,7 @@ protected int nextBatchId() {
 
   public void cleanup() {
     if (!inflightBatchesPerAddress.isEmpty()) {
+      logger.warn("Clear " + this.getClass().getSimpleName());
```
Comment (pan3793) [inline common/src/main/java/org/apache/celeborn/common/write/InFlightRequestTracker.java:179]:
```suggestion
      logger.warn("Clear {}", this.getClass().getSimpleName());
```

**Review by pan3793 — APPROVED** [review]

**Review by RexXiong — APPROVED** [review]
LGTM!

**Pull request closed without merging**